### PR TITLE
Display more detailed leaderboard on review page

### PR DESCRIPTION
**Pull request by MYKatz**
Added a separate leaderboard, counting the last seven days' worth of reviews. "fix" for #148 

screenshot-localhost_9000-2019 11 09-23_56_29

### CODE REVIEW

**Comment by MYKatz** [comment]
Oh fuck gotta update some tests

**Comment by theopolisme** [comment]
This is gonna be awesome!!! :D Can we do a single table with two columns though (maybe something like "all" and "7d")?

**Comment by MYKatz** [comment]
> This is gonna be awesome!!! :D Can we do a single table with two columns though (maybe something like "all" and "7d")?

That's a great idea. Will do

**Comment by MYKatz** [comment]
Alright @theopolisme, looks to be working nicely now on my end.
screenshot-localhost_9000-2019 11 10-00_56_20

File: src/Review/Review.tsx [hunk 1]
```diff
@@ -158,9 +161,15 @@ class Review extends React.Component<IReviewProps, IReviewComponentState> {
 				<div className="container left-sidebar-content">
 					<table className="table treehacks-body-text">
 						<tbody>
+							<tr>
+								<th></th>
```
Comment (theopolisme) [inline src/Review/Review.tsx]: Let's make a header "Reviewer"? To add some visual balance

File: backend/routes/user_review.ts [hunk 2]
```diff
@@ -12,10 +12,20 @@ import { injectDynamicApplicationContent } from "../utils/file_plugin";
...
     Application.aggregate([
         { $match: { reviews: { "$exists": 1 } } },
         { $unwind: "$reviews" },
-        { $group: { _id: "$reviews.reader.email", count: { $sum: 1 } } },
+        { $group: { _id: "$reviews.reader.email", count: { $sum: 1 }, dates: { $push: "$reviews.reviewedAt"} } },
+        { $project: {
+            _id: "$_id",
+            count: "$count",
+            recentCount: { //last seven days
+                $size: { $filter: { input: "$dates", as: "d", cond: { $gt: [ "$$d", new Date(new Date().setDate(new Date().getDate()-7)) ] } } }
```
Comment (theopolisme) [inline backend/routes/user_review.ts]: Can't you remove this `new Date(new Date().setDate(new Date().getDate()-7))` and simply use `d` (defined above) instead? (Maybe you were going for that but didn't finish refactoring?)

**Review by theopolisme — CHANGES_REQUESTED** [review]
Awesome!! Minor tweaks and then let's get this merged ❤️

**Comment by MYKatz** [comment]
@theopolisme made the tweaks. Next time I'll work on being more careful lol

**Comment by theopolisme** [comment]
🎉 Awesome work!!!

**Pull request merged**